Makefile

```diff
@@ -484,7 +485,8 @@ PLATFORM_LINKER_FLAGS = $(BUN_CFLAGS) \
 		-static-libgcc \
 		-fno-omit-frame-pointer \
 		-Wl,--compress-debug-sections,zlib \
-		-l:libatomic.a \
+		-Bstatic \
```

Comment:
do we need to un-static it? IIRC, this flag says "make everything after this static" and we only want this one library statically linked

Reply:
alright. i was under the impression everything here was static since i saw `-static-libstdc++` and `STATIC_MUSL_FLAG` but if thats not the case then we do have to unstatic it